Backend/passport-config.js

```diff
@@ -0,0 +1,40 @@
+// const passport = require('passport');
```

Comment:
@SemanBoon Could you please clean up the code that has already been commented out, or simply add a comment/todo to explain why you are keeping those?

Reply:
They are all base code I need to work on JWT tokens for my stretch feature, I'll add a comment on it and ensure that I clean everything up before creating my next PR.
Thank you bo.

Reply:
Cool, just add what you're saying as a comment! Thanks